test/py/test_vfu_realize_ctx.py

```diff
@@ -81,6 +92,8 @@ def test_vfu_realize_ctx_pci_bars():
     hdr = get_pci_header(ctx)
     assert hdr.bars[0].io == 0x1
     assert hdr.bars[1].io == 0
+    assert hdr.bars[2].io == 0x4
```

Comment:
I know :93 was already like this, but do you mind adding constants for these instead of bare values?